test/TestAll.cpp

```diff
@@ -0,0 +1,13 @@
+#include <gtest/gtest.h>
+#include "CollisionDetectorTest.cpp"
+#include "GameObjectTest.cpp"
+#include "GameStateHandlertest.cpp"
+#include "HitboxTest.cpp"
+#include "KeyboardListenerTest.cpp"
+#include "MissileLauncherTest.cpp"
```

Comment:
W/o the includes, i get:
[==========] Running 0 tests from 0 test cases.
[==========] 0 tests from 0 test cases ran. (0 ms total)
[  PASSED  ] 0 tests.

Not sure why, but when we used a normal makefile, it'd discover the tests, but w/ CMake, it seems like i need the includes for these. 

Example where they do it: https://www.eriksmistad.no/getting-started-with-google-test-on-ubuntu/